scripts/docker/rebuild_test_astrobee_kinetic.Dockerfile

```diff
@@ -6,8 +6,8 @@ FROM astrobee/astrobee:latest-kinetic
 # Rebuild the code
 COPY . /src/astrobee
 RUN /src/astrobee/scripts/configure.sh -l -F -D -p /opt/astrobee -b /build/astrobee
-RUN cd /build/astrobee && make install -j`nproc`
+RUN cd /build/astrobee && make install -j6
 COPY ./astrobee/resources /opt/astrobee/share/astrobee/resources
 
 # Run tests
-RUN cd /build/astrobee && make test && make run_tests
\ No newline at end of file
+RUN cd /build/astrobee && make -j6 tests && make -j6 test
```

Comment:
Could you explain the division between astrobee_base -> astrobee -> rebuild_test_astrobee?

Why not do astrobee (same as it used to be) -> test_astrobee? Then in test_astrobee just do make tests && make test?

Reply:
I didn't think that we could chain the docker images in the github action, I'll try! That would be better yes.

Reply:
Done! Thanks!